src/vs/workbench/services/textfile/electron-browser/textFileService.ts

```diff
@@ -55,20 +56,73 @@ export class TextFileService extends AbstractTextFileService {
 	}
 
 	public resolveTextContent(resource: URI, options?: IResolveContentOptions): TPromise<IRawTextContent> {
-		return this.fileService.resolveStreamContent(resource, options).then(streamContent => {
-			return ModelBuilder.fromStringStream(streamContent.value).then(res => {
-				const r: IRawTextContent = {
-					resource: streamContent.resource,
-					name: streamContent.name,
-					mtime: streamContent.mtime,
-					etag: streamContent.etag,
-					encoding: streamContent.encoding,
-					value: res.value,
-					valueLogicalHash: res.hash
-				};
-				return r;
+
+		const result: IRawTextContent = {
+			resource: undefined,
```

Comment:
What a type is `undefined`???
Are you sure that you need this value which will be always `undefined` in your program runtime